src/sentry/migrations/0470_denormalize_user_is_active.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.28 on 2023-05-30 17:29
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0469_backfill_orgmembermapping"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                    ALTER TABLE "sentry_organizationmember" ADD COLUMN "user_is_active" BOOLEAN NOT NULL DEFAULT TRUE;
+                    """,
+                    reverse_sql="""
+                    ALTER TABLE "sentry_organizationmember" DROP COLUMN "user_is_active";
+                    """,
+                    hints={"tables": ["sentry_organizationmember"]},
```

Comment:
I'm curious why the auto-generated migration isn't sufficient to add this column

Reply:
Good question, it's covered here https://develop.sentry.dev/database-migrations/#adding-columns-with-a-default